koptional/src/main/kotlin/com/gojuno/koptional/Optional.kt

```diff
@@ -24,7 +24,7 @@ sealed class Optional<out T : Any> {
          * prefer using the [toOptional][com.gojuno.koptional.toOptional] extension function.
          */
         @JvmStatic
-        fun <T : Any> toOptional(t: T?): Optional<T> = t.toOptional()
+        fun <T : Any> toOptional(t: T?): Optional<T> = if (t == null) None else Some(t)
```

Comment:
heh, parameter name `t` is part of public API in Kotlin, if you can come up with something more meaningful but short that'd be nice

I have `obj` on mind, but idk

Reply:
Good catch! What about `value`?

Reply:
Much better, let's use that!

Reply:
aa8b028